.webpack/webpack.lantern.base.js

```diff
@@ -86,6 +86,10 @@ module.exports = (env, argv, { SRC_DIR, DIST_DIR }) => {
           __dirname,
           '../node_modules/dom-helpers/class/addClass'
         ),
+        // 'dom-helpers/removeClass': path.resolve(
+        //   __dirname,
+        //   '../node_modules/dom-helpers/class/removeClass'
+        // ),
```

Comment:
What was this for?

Reply:
I had some webpack issues. This is something I originally added so that I could run the PAC locally. Then I realized I pushed it in the PR. So I deleted it again. Sorry for the confusion!